.github/workflows/ci.yaml

```diff
@@ -1,79 +1,87 @@
-on:
+'on':
   push:
     branches:
-    - master
```

Comment:
what about the `master` branch? Don't we still need to run these ci workflows on the master branch?

Reply:
This PR is merging into java7 branch. it won't affect the ci in master.